Gemfile.lock

```diff
@@ -420,6 +420,10 @@ GEM
     kaminari (0.17.0)
```

Comment:
Should kaminari still be pinned to 0.17.0? I noticed that there were breaking changes in the pager methods (`#num_pages` becamse `#total_pages`). The breaks will only be noticed if there is direct interaction with the pager via Hydra code.

https://github.com/ndlib/sipity/commit/ab419af73a1662030fa01ce621cf5e20687e0c8b